### DISCUSSION TITLE
Documentation pointing to src/main.js

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [skeletonlabs/skeleton#902] [General]
So following the manual install I found that there is, what I believe it's an issue on the Themes installation.

https://www.skeleton.dev/guides/themes

In that page it mentions:

> Import your desired preset into `/src/main.js` before your global stylesheet.

I believe that should be `/src/routes/+layout.svelte` if I'm not mistaken.

**Comment by endigo9740** [comment]
You're correct @MrAmericanMike, we've already caught and resolved the issue. It'll go live as part of our release next week. Sorry for the trouble!
